CHANGELOG.md

```diff
@@ -1,10 +1,31 @@
 # Changelog
 
-## 4.0.1
+All notable changes to this project will be documented in this file.
+Each new release typically also includes the latest modulesync defaults.
+These should not affect the functionality of the module.
 
-Change Forge org to `puppet` (project moved from herculestem to Vox Pupuli)
+## [v5.0.0](https://github.com/voxpupuli/puppet-augeasproviders_ssh/tree/v5.0.0) (2022-08-29)
 
-## 4.0.0
+[Full Changelog](https://github.com/voxpupuli/puppet-augeasproviders_ssh/compare/4.0.0...v5.0.0)
+
+**Breaking changes:**
+
+- Drop Non LTS Ubuntu 18.10/19.04/19.10; Add 22.04 [\#78](https://github.com/voxpupuli/puppet-augeasproviders_ssh/pull/78) ([bastelfreak](https://github.com/bastelfreak))
+- Drop Eol Debian 7/8/9; Add Debian 10/11 support [\#77](https://github.com/voxpupuli/puppet-augeasproviders_ssh/pull/77) ([bastelfreak](https://github.com/bastelfreak))
+- Drop RHEL 6 support [\#76](https://github.com/voxpupuli/puppet-augeasproviders_ssh/pull/76) ([bastelfreak](https://github.com/bastelfreak))
+
+**Implemented enhancements:**
+
+- Convert to PDK [\#62](https://github.com/voxpupuli/puppet-augeasproviders_ssh/pull/62) ([raphink](https://github.com/raphink))
```

Comment:
I think we should skip-changelog on this issue because it is not relevant?

Reply:
done